internal/logger/console.go

```diff
@@ -222,22 +222,106 @@ func (i errorMsg) pretty(msg string, args ...interface{}) {
 	if msg == "" {
 		fmt.Fprintln(Output, args...)
 	} else {
-		fmt.Fprintf(Output, msg, args...)
+		fmt.Fprintf(Output, `ERRO: `+msg, args...)
```

Comment:
typo?

Reply:
No it is for alignment @kannappanr just like how logrus does it. 
![image](https://github.com/user-attachments/assets/453dab09-a3ab-49ff-9890-5b7171b1431e)

FATA is not a typo, its implicit that its FATAL